src/Ting/Driver/Mysqli/Result.php

```diff
@@ -144,6 +145,9 @@ protected function format($data)
                         // These case is here as a reminder
                         $value = (string)$value;
                         break;
+                    case MYSQLI_TYPE_GEOMETRY:
```

Comment:
instead of implemeting this in Mysqli/Result, may you implement it in `CCMBenchmark\Ting\Repository\Metadata`, methods `getEntityProperty` and `setEntityProperty` to have serialization and unserialization supported.